### ISSUE TITLE
Backups in summary erroneously reported as deleted

### ISSUE THREAD

**Issue by HaleTom** [digint/btrbk#183]
Below in the summary, 5 backups cannot be deleted (RO filesystem), but they are all listed with the `---` prefix, meaning deleted.

Only one summary error message appears, possibly leading someone reading only the summary information to conclude that 4 were deleted and only the last had an issue.

```
$ sudo btrbk clean -v rootfs
btrbk command line client, version 0.25.1  (Sat Sep 16 14:24:05 2017)
Using configuration: /etc/btrbk/btrbk.conf
Using transaction log: /var/log/btrbk.log
Cleaning incomplete backups in: /media/backup/btrbk/rootfs/rootfs.*
[delete] target: /media/backup/btrbk/rootfs/rootfs.20170915T2200
[delete] target: /media/backup/btrbk/rootfs/rootfs.20170915T2300
[delete] target: /media/backup/btrbk/rootfs/rootfs.20170916T0821
[delete] target: /media/backup/btrbk/rootfs/rootfs.20170916T0900
[delete] target: /media/backup/btrbk/rootfs/rootfs.20170916T1000
ERROR: cannot delete '/media/backup/btrbk/rootfs/rootfs.20170915T2200': Read-only file system
ERROR: cannot delete '/media/backup/btrbk/rootfs/rootfs.20170915T2300': Read-only file system
ERROR: cannot delete '/media/backup/btrbk/rootfs/rootfs.20170916T0821': Read-only file system
ERROR: cannot delete '/media/backup/btrbk/rootfs/rootfs.20170916T0900': Read-only file system
ERROR: cannot delete '/media/backup/btrbk/rootfs/rootfs.20170916T1000': Read-only file system
ERROR: Failed to delete btrfs subvolumes: /media/backup/btrbk/rootfs/rootfs.20170915T2200 /media/backup/btrbk/rootfs/rootfs.20170915T2300 /media/backup/btrbk/rootfs/rootfs.20170916T0821 /media/backup/btrbk/rootfs/rootfs.20170916T0900 /media/backup/btrbk/rootfs/rootfs.20170916T1000
Completed within: 0s  (Sat Sep 16 14:24:05 2017)
--------------------------------------------------------------------------------
Cleanup Summary (btrbk command line client, version 0.25.1)

    Date:   Sat Sep 16 14:24:05 2017
    Config: /etc/btrbk/btrbk.conf
    Filter: group=rootfs

Legend:
    ---  deleted subvolume (incomplete backup)
--------------------------------------------------------------------------------
/media/backup/btrbk/rootfs/rootfs.*
--- /media/backup/btrbk/rootfs/rootfs.20170915T2200
--- /media/backup/btrbk/rootfs/rootfs.20170915T2300
--- /media/backup/btrbk/rootfs/rootfs.20170916T0821
--- /media/backup/btrbk/rootfs/rootfs.20170916T0900
--- /media/backup/btrbk/rootfs/rootfs.20170916T1000
!!! Target "/media/backup/btrbk/rootfs" aborted: Failed to delete incomplete target subvolume
$ 
```

**Comment by HaleTom** [comment]
I realise that you don't have much time to spend on btrbk at the moment.

I just wanted to say that this tool is *amazing* and I'd be really stuck without it. I really appreciate the quality of development which is evident in the code.

**Comment by digint** [comment]
Fixed in 6c2d56d6214e6ade4041d8a1a6053ca1bb568db7

Thanks for reporting!
